shared/constants/snaps/permissions.ts

```diff
@@ -25,6 +26,9 @@ export const ExcludedSnapPermissions = Object.freeze({
 });
 
 export const ExcludedSnapEndowments = Object.freeze({
+  // Move to below fence once implemented
+  'endowment:name-lookup':
+    'This permission is still in development and therefore not available.',
```

Comment:
Why can't we move this now?

Reply:
IMO it should be disabled until we add an actual integration for it